### DISCUSSION TITLE
How to temporarily disable format_on_save for current file

### DISCUSSION THREAD

**Discussion by c02y** [LunarVim/LunarVim#2420] [Q&A] [answered]
Sometimes I want to edit a file which is not formatted by the original author, I don't want it to be formatted on save since it makes a lot of more changes to the file once it is formatted. How can temporarily disable format_on_save for it?

**Comment by abzcoding** [comment, marked as answer]
```vim
:LvimToggleFormatOnSave
```

**Reply by c02y** [reply]
Thanks, didn't find it in any doc about format, maybe doc needs update?

**Reply by abzcoding** [reply]
yeah, PRs would be appreciated 🥺
